SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -3,6 +3,17 @@ import SSA.Projects.InstCombine.ForMathlib
 import SSA.Projects.InstCombine.LLVM.Semantics
 import Mathlib.Tactic
 
+#check BitVec.sshiftRight
+
+-- a3 a2 a1 a0 >> 2
+-- 0 0 a3 a2  [0]
+lemma BitVec.getLsb_sshiftRight (a : BitVec w) (s : Nat) : (a.sshiftRight s).getLsb i =
+   if (i + s ≥ w) then (a.getMsb 0) else a.getLsb (s + i) :=
+  by sorry
+-- bitvec_AddSub_1152
+
+#print propext
+
```

Comment:
```suggestion
```

Let's focus on one lemma per PR, in this case  `add_eq_xor`. If we drop this wip-code, we can already push this PR and have one more theorem proven. If we do this, the PR of this title also should be updated.